### PR TITLE
Add RV64GC target

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks fine to me. How have you testing this?

File: wasm_bindgen/3rdparty/crates/defs.bzl [hunk 1]
```diff
@@ -391,7 +391,7 @@ _CONDITIONS = {
...
-    "cfg(not(windows))": ["aarch64-apple-darwin", "aarch64-apple-ios", "aarch64-apple-ios-sim", "aarch64-linux-android", "aarch64-unknown-linux-gnu", "arm-unknown-linux-gnueabi", "armv7-linux-androideabi", "armv7-unknown-linux-gnueabi", "i686-apple-darwin", "i686-linux-android", "i686-unknown-freebsd", "i686-unknown-linux-gnu", "powerpc-unknown-linux-gnu", "riscv32imc-unknown-none-elf", "s390x-unknown-linux-gnu", "wasm32-unknown-unknown", "wasm32-wasi", "x86_64-apple-darwin", "x86_64-apple-ios", "x86_64-linux-android", "x86_64-unknown-freebsd", "x86_64-unknown-linux-gnu"],
+    "cfg(not(windows))": ["aarch64-apple-darwin", "aarch64-apple-ios", "aarch64-apple-ios-sim", "aarch64-linux-android", "aarch64-unknown-linux-gnu", "arm-unknown-linux-gnueabi", "armv7-linux-androideabi", "armv7-unknown-linux-gnueabi", "i686-apple-darwin", "i686-linux-android", "i686-unknown-freebsd", "i686-unknown-linux-gnu", "powerpc-unknown-linux-gnu", "riscv32imc-unknown-none-elf", "riscv64gc-unknown-none-elf" "s390x-unknown-linux-gnu", "wasm32-unknown-unknown", "wasm32-wasi", "x86_64-apple-darwin", "x86_64-apple-ios", "x86_64-linux-android", "x86_64-unknown-freebsd", "x86_64-unknown-linux-gnu"],
```
Comment (ted-logan) [inline wasm_bindgen/3rdparty/crates/defs.bzl]: Awkward hard-to-find typo here, there's a missing `,` after `"riscv64gc-unknown-none-elf"`

**Comment by furquan-rivos** [comment]
Change looks okay apart from the comment that Ted made. However, this is the first time I am looking at this package, so I might not be familiar with any other things that are required. Can you please have Tomasz look at this change as well?

**Comment by furquan-rivos** [comment]
> Change looks okay apart from the comment that Ted made. However, this is the first time I am looking at this package, so I might not be familiar with any other things that are required. Can you please have Tomasz look at this change as well?

I was unable to add Tomasz directly here as a reviewer since I couldn't find him in the list.

**Review by ted-logan — APPROVED** [review]
I have not successfully used these rules to cross-compile riscv64 for Rust, but it seems like a good step forward at least.

**Comment by ghost** [comment]
In my testing with these changes, specifying `-cpu=riscv64` on the bazel command line, I no longer see RV64-related errors on my targets. I see numerous _other_ problems on my test build target, but certainly nothing more about toolchain selection.

**Pull request merged**